docs/Developers/Newcomers_Guide.rst

```diff
@@ -164,6 +164,17 @@ git/GitHub, we strongly recommend following a tutorial, such as `this one
 We also have a `page dedicated to git commands <http://coala.io/git>`_ that
 will help you learn the basics.
 
+.. note::
+
+    We also have some of our repositories on
+    `Gitlab <https://gitlab.com/coala>`_.
+    Gitlab contributions also count for the promotion of a
+    newcomer to developer.
+
+We also have a
+`wiki page <https://github.com/coala/coala/wiki/GitLab>`_
+that will illustrate more about Gitlab.
```

Comment:
This seems to kinda refer to the note, which is optional and doesn't count to the "main-text". If GitLab was only mentioned in a note, you aren't allowed to reference it here, that's confusing.

Maybe this piece of text fits better if you move it upwards a bit?